.github/workflows/ci.yaml

```diff
@@ -29,3 +29,50 @@ jobs:
         run: |
           flit install -s
           pytest tests/unit
+
+  linux-integration_tests-sqlserver:
+    name: "Linux - integration tests - Python ${{ matrix.PYTHON_VERSION }} - SQL server with ${{ matrix.DRIVER }} driver"
+    runs-on: ubuntu-20.04
+    env:
+      CI: True
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ['3.8']
+        DRIVER: ["mssql"] #, "mssql-freetds"]
+    services:
+      DB:
+        image: mcr.microsoft.com/mssql/server:2019-latest
+        env:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: pytsql-Test-123

```

Comment:
Can we set this to `QuantCo@MSSQL`? That way it would match the password from `start_mssql.sh`. As is, the changes in `conftest` don't allow me to run the integration tests locally.

Reply:
What if we change the password in the script? Seems odd to me to have `QuantCo` as part of a password.

Reply:
I have no opion wrt what the password is as long is it's the same one for ci and the local docker container.